texttospeech/api/QuickStart/Program.cs

```diff
@@ -0,0 +1,74 @@
+﻿/*
+ * Copyright (c) 2018 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+// [START tts_quickstart]
+
+using System;
+using System.IO;
+using Google.Cloud.TextToSpeech.V1;
+
+
+public class QuickStart
+{
+    /// <summary>
+    /// Synthesizes speech from the input string of text
+    /// </summary>
+    /// <param name="args">The command-line arguments.</param>
```

Comment:
Remove this whole comment UNLESS we use the same comment on other Quickstarts

If this looked exactly like the other Quickstarts, LGTM

I personally would advocate to remove all 4 comment LOC. They are a huge waste of valuable vertical space in the embedded code snippet & there is no such thing as a C# programmer who doesn't know what `public static void Main(string[] args)` means, it's literally what they teach the first day of CS 101

Reply:
( we don't use this comment in the Quickstart for Video or Speech, I didn't snoop any further )

Minor thing, but it's awesome when Quickstarts are short & sweet :)

Reply:
Done